projects/packages/my-jetpack/_inc/hooks/use-purchases/index.js

```diff
@@ -7,5 +7,8 @@ import { STORE_ID } from '../../state/store';
  * @returns {object} site purchases data
  */
 export default function usePurchases() {
-	return useSelect( select => select( STORE_ID ).getPurchases() );
+	return {
+		purchases: useSelect( select => select( STORE_ID ).getPurchases() ),
+		isFetchingPurchases: useSelect( select => select( STORE_ID ).isRequestingPurchases() ),
+	};
```

Comment:
This is more of a personal preference, so feel free to ignore. I like to group things a bit differently:

```suggestion
	const { purchases, isFetchingPurchases } = useSelect( select => {
		const { getPurchases, isRequestingPurchases } = select( STORE_ID );

		return {
			purchases: getPurchases(),
			isFetchingPurchases: isRequestingPurchases(),
		};
	} );

	return {
		purchases,
		isFetchingPurchases,
	};
```